src/logic/utils.ts

```diff
@@ -1,3 +1,38 @@
+import { PLAYER_MINIMUM_GAMES_REQUIRED, PLAYER_MINIMUM_WINS_REQUIRED } from "../components/constants";
+import { Match } from "../types/domain/Match";
+import { Player } from "../types/domain/Player";
+
 export function getWinRatePercentage(winCount: number, totalCount: number) {
     return totalCount > 0 ? Math.round((winCount / totalCount) * 100) : 0;
 }
+
+export function avgWinTurn(player: Player) {

```

Comment:
i'm going to be annoying here :) 

typically I prefer naming conventions that obtain a value from a set of parameters to start with "get". in this case, I would do getAverageWinTurn. 

I don't like abbreviating "average" to "avg" personally since we want to be as specific in our method names as possible (usually)